.nancy-ignore

```diff
@@ -1,4 +1,4 @@
 # Ignore CVE fixed in prometheus 2.7.1 as the tag version is using a newer prometheus version
 CVE-2019-3826
 # Ignore CVE jwt-go before 4.0.0-preview1 allows attackers to bypass intended access restrict...https://github.com/dgrijalva/jwt-go/issues/290
-CVE-2020-26160 until=2021-06-01
+CVE-2020-26160
```

Comment:
This was fixed in master. I finally merged https://github.com/giantswarm/prometheus-meta-operator/pull/538

Reply:
Let me rebase master then :)